.github/workflows/ci.yaml

```diff
@@ -25,9 +25,9 @@ jobs:
 
   build-and-push-container-images:
     name: Build and push container images
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
```

Comment:
Why the change to a lower version ?
Ubuntu 22 to Ubuntu 20 ? 
Something python related?

Reply:
Yes.
https://github.com/trento-project/web/pull/2713